docs/about/whats_new.rst

```diff
@@ -3,36 +3,39 @@
 What's New
 ==========
 
-v1.2.3
+v1.3.0
 ------
 
  - Updated the Postgres product views to include the whole dataset metadata document.
 
- - `init` now recreates the product views by default every time it is run.
+ - ``init`` now recreates the product views by default every time it is run.
```

Comment:
expanding this out to 
```
:code:`datacube system init`
OR
``datacube system init``
```
would be more readable. (I'm not sure how the two options render.)
I looked into included links to datacube command documentation, but it wasn't trivial so I've left it for now.